superdesk/publish/publish_content.py

```diff
@@ -96,54 +96,76 @@ def get_queue_items(retries=False):
     return get_resource_service(PUBLISH_QUEUE).get(req=request, lookup=lookup)
 
 
-@celery.task(soft_time_limit=1800, bind=True)
+@celery.task(soft_time_limit=60, bind=True)
 def transmit_subscriber_items(self, queue_items, subscriber):
     # Attempt to obtain a lock for transmissions to the subscriber
     lock_name = get_lock_id("Subscriber", "Transmit", subscriber)
     if not lock(lock_name, expire=300):
```

Comment:
Yes, one worker becomes blocked and after 300s the lock has been released and other worker becomes blocked and so on until all workers are blocked for 30 minutes. This PR solves the block issue with one individual worker(current timeout is 10s for every transmission) and also solves the lock issue because now the lock release is 300s and the timeout for worker that request lock is 60.